scripts/kconfig/kconfiglib.py

```
@@ -1545,8 +1545,7 @@ def write_config(self, filename=None, header=None, save_old=True,
           written.
 
           Errors are silently ignored if <filename>.old cannot be written (e.g.
-          due to being a directory, or <filename> being something like
-          /dev/null).
+          due to being a directory).
```

Maybe this:

```suggestion
          due to permissions errors.)
```

It's not true that errors are ignored if the file is a directory, because `os.path.isfile` returns `False` for directories (at least on my computer), so the attempt to write a directory is never made.

---

Here we're referring <filename>.old, not <filename> which was checked by os.path.isfile() in my change.

Anyway, this change you suggested could be less confusing, and therefore could be better of what
I changed in that commit.

Should I do the pull request now, or wait for this to get its final form, so the pull request could
have exactly same commit message contents and exactly same change?

---

hmm, forget my above comment about `<filename>.old` and `<filename>` ... I must have been too
tired to examine the content well enough. The other issue (kconfig pull request) got my attention there...

---

Ok, The initial changes in comments were just to do minimal potentially good enough change to
the related comments. The changes in line 1548 was made to match the change in comment lines
in 6420 (6424 after change). 

Now I looked whether the above change could be done w/o the change in 6424 (or should that be
changed also). 

To me it looks like the change in 1548 can be made w/ change in 6424.

I almost pressed the 'Commit suggestion' button, but then noticed that perhaps `s/.)/)./` there (i.e.
trailing perions after closing parentheses). Perhaps I accept, then re-edit (if necessary)